lib/Ceilometer/Types/Base.hs

```diff
@@ -0,0 +1,24 @@
+module Ceilometer.Types.Base where
+
+import           Control.Isomorphism.Partial.Iso
+import           Data.Bimap                      (Bimap)
+import qualified Data.Bimap                      as B
+import           Data.Word
+
+-- |Iso generator
+generateIsoFromBimap :: (Ord a, Ord b) => Bimap a b -> Iso a b
```

Comment:
Well, the bimap is a sort of bijection, if you restrict  image to be only the things in the map.